rudder-core/src/main/scala/com/normation/rudder/services/queries/LdapQueryProcessor.scala

```diff
@@ -205,13 +205,17 @@ class PendingNodesLDAPQueryChecker(
 ) extends QueryChecker {
 
   override def check(query:Query, limitToNodeIds:Seq[NodeId]) : Box[Seq[NodeId]] = {
-    for {
-      entries <- checker.internalQueryProcessor(query, Seq("1.1"), Some(limitToNodeIds))
-      ids <- sequence(entries) { entry =>
-        checker.ldapMapper.nodeDn2OptNodeId(entry.dn) ?~! "Can not get node ID from dn %s".format(entry.dn)
+    if(query.criteria.isEmpty) {
+      Full(Seq.empty[NodeId])
```

Comment:
shouldn't it log a warning or something ?

Reply:
I can add a log.debug, yes